examples/encrypt.rs

```diff
@@ -12,12 +12,11 @@ async fn main() -> Result<()> {
 
     // Initialize a channel with cipher parameters so that published messages
     // get encrypted.
-    let cipher_key = ably::crypto::generate_random_key::<ably::crypto::Key256>();
-    let params = ably::rest::CipherParams::from(cipher_key);
+    let cipher = ably::crypto::Cipher::default();
```

Comment:
`ChipherBuilder::new().len(KeyLen::Bits128).build()`

Reply:
> so here what is the default key size?

Whatever the spec says!

Reply:
> ChipherBuilder::new().len(KeyLen::Bits128).build()

Ah ok, so given I'm building a cipher mode and not a key, perhaps the builder function should be named `keyLength`?

> > so here what is the default key size?

> Whatever the spec says!

Ok, I think it's worth documenting what the default key size is on the `default()` method, since users are more likely to be reading the ably-rust docs, not the spec 🙂

Reply:
yes len -> key_len sounds fine. Documentation too.